docs/getting_started.md

```diff
@@ -267,6 +267,9 @@ llama stack build --config llama_stack/distribution/templates/local-ollama-build
 
 #### How to build distribution with Docker image
 
+> [!TIP]
+> Podman is supported as an alternative to Docker. Set `DOCKER_BINARY` to `podman` in your environment to use Podman.
```

Comment:
Yes, in fact we use podman internally as well.

Reply:
@ashwinb great! What would you think of using `container` instead of `docker` as the type to make it a bit more generic?  similar to `Containerfile` instead of `Dockerfile` ...